src/components/add-update-EventPage/AddEventPage.jsx

```diff
@@ -0,0 +1,196 @@
+import React, { useState } from 'react'
+import './AddEventPage.css'
+import {postEvent} from './AddEvent.js'
+
+
+
+
+export default function AddEventPage() {
+
+
+    const [event_title,setevent_title]=useState("");
+    const [imgUrl,setimgUrl]=useState("");
+    const [maxParticipants,setmaxParticipants]=useState(0);
+    const [location,setlocation]=useState("");
+   const [formsummary, setFormsummary] = useState({
+    summary: '',
+    description: '',
+  });
+   const [formDate, setFormDate] = useState({
+    date: '',
+    event_start_time: '',
+  });
+  const [items, setItems] = useState([]);
```

Comment:
since "items" is considered part of the event info maybe we can move this state inside the event object state